whenchanged/whenchanged.py

```diff
@@ -191,7 +209,17 @@ def main():
         if verbose:
             print("When '%s' changes, run '%s'" % (files[0], print_command))
 
-    wc = WhenChanged(files, command, recursive, run_once, run_at_start)
+    return WhenChanged(files, command, recursive, run_once, run_at_start)
```

Comment:
Would prefer if `parse_args()` simply returned the list of parsed arguments and didn't create a `WhenChanged` object. i.e. just return a tuple `(files, command, recursive, run_once, run_at_start)` which can later be passed to `WhenChanged()`